apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -155,47 +182,46 @@ const OnboardingPage = ({
   });
 
   const handleSelectAccount = async (teamId?: number) => {
-    try {
-      setIsSelectingAccount(true);
-      if (appMetadata.isOAuth) {
-        const state = JSON.stringify({
-          appOnboardingRedirectUrl: getAppOnboardingRedirectUrl(appMetadata.slug, teamId),
-          teamId,
-        });
-
-        const res = await fetch(
-          `/api/integrations/${
-            appMetadata.slug == "stripe" ? "stripepayment" : appMetadata.slug
-          }/add?state=${state}`,
-          {
-            method: "GET",
-            headers: {
-              "Content-Type": "application/json",
-            },
+    mutation.mutate(
+      {
+        isOmniInstall: true,
```

Comment:
setting isOmniInstall to true so that we can be redirected back to /apps/installation/[step]?slug
if not set it redirects to apps/installed/[category] and the redirection can't be controlled

Reply:
I wonder if we still need this prop. All instances are set to true.

Reply:
Fair point @joeauyeung

Reply:
working on a refactor in #15550

Reply:
pushed a refactor in 

https://github.com/calcom/cal.com/pull/15193/commits/b2f478cdb68ab0c5caf30c581e10547548bafbbd and https://github.com/calcom/cal.com/pull/15193/commits/69378039ec5df39cd7e74d39bf4c3fd148ac537b